airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/AirbyteType.kt

```diff
@@ -49,7 +49,11 @@ data object ArrayTypeWithoutSchema : AirbyteType() {
     override val isArray = true
 }
 
-data class ObjectType(val properties: LinkedHashMap<String, FieldType>) : AirbyteType() {
+data class ObjectType(
+    val properties: LinkedHashMap<String, FieldType>,
+    val additionalProperties: Boolean? = null,
+    val required: List<String> = emptyList<String>()
```

Comment:
@edgao, @frifriSF59 , I think we've been back and forth on the required/nullable concept in the typing of the bulk CDK. Any concerns with this?

Reply:
I think this is already represented on the FieldType? (i.e. nullable vs not)

Reply:
For me, required and nullable are two different things. For example `{"id": 1, "toto": null, "tata": "a value"}` would mean that we want to update both toto and tata. `{"id": 1, "tata": "a value"}` would only update tata. In this case, toto is both nullable and not required